recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+export DYLD_LIBRARY_PATH=$PREFIX/lib
+
+# https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg11423.html
+export LD_LIBRARY_PATH=$PREFIX/lib
+
+./configure \
```

Comment:
Really hoped GitHub would handle that link better. Anyways, he was seeing segfaults when using `hdf5`/`h5py` with Python when `hdf5` was built with `cmake`. I prefer using `configure` to segfaults any day. If this situation is any bit similar here, then we would be wise to stick with what works.